packages/suite/src/actions/wallet/selectedAccountActions.ts

```diff
@@ -158,6 +167,7 @@ const getAccountState = (state: AppState): SelectedAccountStatus => {
                     network,
                     discovery,
                     params,
+                    mode: !isTorEnabled ? ['tor-disable'] : mode,
```

Comment:
Should we merge the `'tor-disable'` mode with the other modes potentially returned from `getAccountStateWithMode`? If not, that's all from me.

Reply:
That is probably the best.
Do you see any problem in returning empty array instead of `undefined` from `getAccountStateWithMode`  ?

Reply:
I think the only difference would be presence of this div with bottom-margin: https://github.com/trezor/trezor-suite/blob/91dd87b7516972da2fec6b87371ed88a39619416/packages/suite/src/components/wallet/AccountMode/index.tsx#L10-L12

But we could change e.g. `!props.mode` to `!props.mode?.length` or something.

Reply:
That sounds reasonable, changes here https://github.com/trezor/trezor-suite/pull/6796/commits/30d1f7350ef9c5858060233b4ce8bbbf921a6a3c